cypress/utils/changePasswordTab.js

```diff
@@ -0,0 +1,12 @@
+export const passwordTab = ()=>{
+    cy.viewport(1280,720);
+     cy.visit('https://spinkart.neetoauth.net')
+     cy.fixture("credentials").then((user) => {
+        cy.login(user.correct.email,user.correct.password);
+      })
+      cy.loginSubmit();
```

Comment:
Here we are just clicking on password tab after visiting profile settings page. We can avoid writing login commands in each function as our first step in login and we have a custom command for login.